e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -35,7 +35,7 @@ const testNonceBump = async (
 	const firstTxHash = await walletClient.sendTransaction({
 		to: "0x00000000000000000000000000000000DeaDBeef",
 		value: 2,
-		gas: 171000,
+		gas: await getIntrinsicGasForFeeCurrency(21000, firstCurrency),
```

Comment:
nit: the value-only intrinsic gas could be defined as a test-wide constant and referenced everywhere